agents/agile_api.py

```
@@ -0,0 +1,150 @@
+
+
+async def _require_auth(request: Request) -> Any:
+    user = getattr(request.state, "user", None)
+    if user is not None:
+        return user
+    auth = request.headers.get("authorization", "")
+    token = auth[7:].strip() if auth[:7].lower() == "bearer " else request.headers.get("x-api-key", "").strip()
+    if token:
+        try:
+            from tokens import verify_token
+            payload = verify_token(token, token_type="access")
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to argument: "token_type"

(S106)